openslides/utils/settings.py.tpl

```diff
@@ -79,6 +79,15 @@ DATABASES = {
 use_redis = False
 
 if use_redis:
+    # Redis configuration for redis-session. Keep this synchronized to the
```

Comment:
typo: 'django-redis-sessions' instead of 'redis-session'

Reply:
Done